backend/live2d_manager.py

```
@@ -0,0 +1,502 @@
+                    animations.append(animation_name)
+            
+            # Default animations if none found
+            if not animations:
+                animations = [
+                    "idle", "talk", "happy", "sad", "surprised", 
+                    "thinking", "greeting", "goodbye", "nod", "shake"
+                ]
+            
+            return animations
+            
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)